README_zh-CN.md

```diff
@@ -181,7 +181,7 @@ MMOCR 是一款由来自不同高校和企业的研发人员共同参与贡献
 扫描下方的二维码可关注 OpenMMLab 团队的 [知乎官方账号](https://www.zhihu.com/people/openmmlab)，加入 OpenMMLab 团队的 [官方交流 QQ 群](https://jq.qq.com/?_wv=1027&k=aCvMxdr3)
 
 <div align="center">
-<img src="resources/zhihu_qrcode.jpg" height="400" />  <img src="resources/qq_group_qrcode.jpg" height="400" />
+<img src="resources/zhihu_qrcode.jpg" height="400" />  <img src="resources/qq_group_qrcode.jpg" height="400" />  <img src="resources/wechat_qrcode.jpg" height="400" />
```

Comment:
There are text descriptions for Zhihu and QQ, we may also need to add a few words for wechat.